### PR TITLE
Update v8 flags

### PR DESCRIPTION
**Pull request by kripken**
Two no longer exist, after I believe they were enabled
by default.

The new staging flag should help us avoid to add many new
specific flags in the future.

### CODE REVIEW

File: scripts/test/shared.py [hunk 1]
```diff
@@ -225,10 +225,9 @@ def has_shell_timeout():
 
 # Default options to pass to v8. These enable all features.
 V8_OPTS = [
+    '--wasm-staging',
```
Comment (aheejin) [inline scripts/test/shared.py:228]: - `--wasm-staging` [includes](https://github.com/v8/v8/blob/ffb4f2b77fd3abe3d7450312aa3c670776d9502d/src/wasm/wasm-feature-flags.h#L32-L71) `mv` and `threads`, so they can be removed
- Bulk memory is shipped and [enabled by default](https://github.com/v8/v8/blob/ffb4f2b77fd3abe3d7450312aa3c670776d9502d/src/wasm/wasm-feature-flags.h#L73-L82), so we can remove that (and the feature flag is scheduled to be removed)
- There is one more flag that is not enabled by default or `--wasm-staging`: [`--experimental-wasm-compilation-hints`](https://github.com/v8/v8/blob/ffb4f2b77fd3abe3d7450312aa3c670776d9502d/src/wasm/wasm-feature-flags.h#L28-L30). Binaryen doesn't use this for anything for now but maybe it'd be better to include this for completeness..?

Comment (kripken) [reply scripts/test/shared.py:229]: Thanks! Updated.

File: scripts/test/shared.py [hunk 2]
```diff
@@ -224,15 +224,13 @@ def has_shell_timeout():
 
 
 # Default options to pass to v8. These enable all features.
+# See https://github.com/v8/v8/blob/master/src/wasm/wasm-feature-flags.h#L73-L82
```
Comment (aheejin) [inline scripts/test/shared.py:227]: This range is only about shipped features. I wonder other info (staged and non-staged features) are relevant too? And also the line numbers are subject to change, unless we take a permalink for it. How about just including the link of a file itself, without line numbers? The whole file is relevant info I think.

Comment (kripken) [reply scripts/test/shared.py:227]: Sounds good.

**Pull request merged**